x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_remote_es.tsx

```diff
@@ -193,91 +201,190 @@ export const OutputFormRemoteEsSection: React.FunctionComponent<Props> = (props)
-                <FormattedMessage
-                  id="xpack.fleet.settings.editOutputFlyout.kibanaAPIKeyLabel"
-                  defaultMessage="Remote Kibana API Key"
-                />
-              }
-              {...inputs.kibanaAPIKeyInput.formRowProps}
-              useSecretsStorage={useSecretsStorage}
-              onToggleSecretStorage={onToggleSecretAndClearValue}
-            >
-              <EuiFieldText
-                fullWidth
-                data-test-subj="kibanaAPIKeySecretInput"
-                {...inputs.kibanaAPIKeyInput.props}
-                placeholder={i18n.translate(
-                  'xpack.fleet.settings.editOutputFlyout.kibanaAPIKeyPlaceholder',
-                  {
-                    defaultMessage: 'Specify Kibana API Key',
-                  }
+                data-test-subj="remoteClusterConfigurationCallout"
+              >
+                {isRemoteClusterInstructionsOpen ? (
+                  <EuiButton onClick={() => setIsRemoteClusterInstructionsOpen(false)}>
+                    <FormattedMessage
+                      id="xpack.fleet.settings.remoteClusterConfiguration.collapseInstructionsButtonLabel"
+                      defaultMessage="Collapse steps"
+                    />
+                  </EuiButton>
+                ) : (
+                  <EuiButton onClick={() => setIsRemoteClusterInstructionsOpen(true)} fill={true}>
+                    <FormattedMessage
+                      id="xpack.fleet.settings.remoteClusterConfiguration.viewInstructionButtonLabel"
+                      defaultMessage="View steps"
+                    />
+                  </EuiButton>
                 )}
-              />
-            </SecretFormRow>
-          ) : (
-            <SecretFormRow
-              fullWidth
-              title={i18n.translate('xpack.fleet.settings.editOutputFlyout.kibanaAPIKeyLabel', {
-                defaultMessage: 'Remote Kibana API Key',
-              })}
-              {...inputs.kibanaAPIKeySecretInput.formRowProps}
-              cancelEdit={inputs.kibanaAPIKeySecretInput.cancelEdit}
-              useSecretsStorage={useSecretsStorage}
-              isConvertedToSecret={isConvertedToSecret.kibanaAPIKey}
-              onToggleSecretStorage={onToggleSecretAndClearValue}
-            >
-              <EuiFieldText
-                data-test-subj="kibanaAPIKeySecretInput"
+                {isRemoteClusterInstructionsOpen && (
+                  <>
+                    <EuiSpacer size="m" />
+                    <FormattedMessage
+                      id="xpack.fleet.settings.remoteClusterConfiguration.description"
+                      defaultMessage="The Remote Elasticsearch output must have additional configuration to sync integrations from this cluster. {documentationLink}."
+                      values={{
+                        documentationLink: (
+                          <EuiLink external={true} href={docLinks.links.fleet.remoteESOoutput}>
```

Comment:
@kilfoyle I expect we'll file a docs request when https://github.com/elastic/kibana/issues/187323 is near completion and I think it probably makes sense for the new content to live on the same remote ES output docs page, but we can update the link here if not.

Reply:
Yup! That makes sense to me. Best to keep all the remote ES stuff together on the same page.